src/components/Footer/__tests__/__snapshots__/Footer.test.tsx.snap

```diff
@@ -1,3 +1,3 @@
8 13.1645 6.156C12.3181 6.06 11.507 6.084 10.79 6.18C10.7284 6.18 10.6758 6.18915 10.6165 6.19945C10.6079 6.20095 10.5991 6.20248 10.5901 6.204C10.1787 6.24 9.17952 6.396 7.92172 6.96C7.48678 7.164 7.22817 7.308 7.22817 7.308C7.22817 7.308 8.20384 6.36 10.3198 5.736L10.2022 5.592C10.2022 5.592 8.59176 5.556 6.89903 6.852C6.89903 6.852 5.20629 9.984 5.20629 13.848C5.20629 13.848 6.19372 15.588 8.7916 15.672C8.7916 15.672 9.22654 15.132 9.57919 14.676C8.08629 14.22 7.52205 13.26 7.52205 13.26C7.52205 13.26 7.6396 13.344 7.85119 13.464C7.86294 13.476 7.8747 13.488 7.89821 13.5C7.91584 13.512 7.93348 13.521 7.95111 13.53C7.96874 13.539 7.98637 13.548 8.00401 13.56C8.29788 13.728 8.59176 13.86 8.86213 13.968C9.34409 14.16 9.92009 14.352 10.5901 14.484C11.4718 14.652 12.5062 14.712 13.6347 14.496C14.1872 14.4 14.7514 14.232 15.3392 13.98C15.7506 13.824 16.2091 13.596 16.691 13.272C16.691 13.272 16.1033 14.256 14.5634 14.7ZM8.67425 11.3997C8.67425 10.6677 9.20323 10.0677 9.87327 10.0677C10.5433 10.0677 11.084 10.6677 11.0723 11.3997C11.0723 12.1317 10.5433 12.7317 9.87327 12.7317C9.21499 12.7317 8.67425 12.1317 8.67425 11.3997ZM12.9649 11.3997C12.9649 10.6677 13.4938 10.0677 14.1639 10.0677C14.8339 10.0677 15.3629 10.6677 15.3629 11.3997C15.3629 12.1317 14.8339 12.7317 14.1639 12.7317C13.5056 12.7317 12.9649 12.1317 12.9649 11.3997Z" fill="currentColor"></path></svg><span class="amplify-visually-hidden">Discord</span></a></li><li><a href="https://github.com/aws-amplify" class="footer-link footer-link--social" rel="noopener noreferrer" target="_blank"><svg class="amplify-icon" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M12.0092 0C5.39329 0 0.0449829 5.49998 0.0449829 12.3042C0.0449829 17.7432 3.47182 22.3472 8.22576 23.9767C8.82013 24.0991 9.03784 23.7119 9.03784 23.3862C9.03784 23.1009 9.01825 22.1232 9.01825 21.1044C5.6901 21.8379 4.99704 19.6377 4.99704 19.6377C4.46219 18.2117 3.6697 17.8452 3.6697 17.8452C2.5804 17.0914 3.74905 17.0914 3.74905 17.0914C4.95737 17.1729 5.59141 18.3544 5.59141 18.3544C6.66087 20.2284 8.38421 19.6989 9.07752 19.3729C9.17645 18.5784 9.4936 18.0284 9.83033 17.7229C7.17589 17.4377 4.38309 16.3784 4.38309 11.6522C4.38309 10.3077 4.85819 9.20771 5.611 8.35222C5.49223 8.04672 5.07615 6.78347 5.73002 5.09273C5.73002 5.09273 6.74022 4.76673 9.01801 6.35572C9.99321 6.08639 10.9989 5.94938 12.0092 5.94823C13.0194 5.94823 14.0492 6.09097 15.0001 6.35572C17.2781 4.76673 18.2883 5.09273 18.2883 5.09273C18.9422 6.78347 18.5259 8.04672 18.4071 8.35222C19.1798 9.20771 19.6353 10.3077 19.6353 11.6522C19.6353 16.3784 16.8425 17.4172 14.1682 17.7229C14.6041 18.1099 14.9803 18.8432 14.9803 20.0044C14.9803 21.6544 14.9607 22.9787 14.9607 23.3859C14.9607 23.7119 15.1786 24.0991 15.7728 23.9769C20.5267 22.3469 23.9535 17.7432 23.9535 12.3042C23.9731 5.49998 18.6052 0 12.0092 0Z" fill="currentColor"></path></svg><span class="amplify-visually-hidden">Github</span></a></li></ul></div></footer></div>"`;
```

Comment:
did this change due to the jest version bump?

Reply:
Yeah the snapshot failed because now it generates it without the escaped \" so I updated the snapshot